src/main/features/first-contact/guards/claimExpiredGuard.ts

```diff
@@ -0,0 +1,17 @@
+import * as express from 'express'
+import { ErrorPaths } from 'first-contact/paths'
+import { User } from 'idam/user'
+
+const logger = require('@hmcts/nodejs-logging').getLogger('first-contact/guards/claimReferenceMatchesGuard')
```

Comment:
change this `claimReferenceMatchesGuard` to match the name of this file

Reply:
Updating now

Reply:
Updated.